importer/validation.py

```diff
@@ -51,6 +51,7 @@ def validate_mandatory_read_fields(spreadsheet):
 def __validate_mandatory_read_fields_for_read(read: RawRead) -> List[str]:
     result = []
     if read.forward_read is None:
+        __validate_row_fill_status(read)
```

Comment:
Understand what you're doing - the `str(read)` will only fail if all of the elements are `None`, so you're checking all of them. I would simplify this, because really, if the forward read is `None`, there's no need to check anything else, and it just causes problems downstream, that you're solving with multiple `is not None and`. Just raise an exception if the forward read is `None`.